compiler/optimizer/OMRTransformUtil.cpp

```diff
@@ -436,3 +436,96 @@ OMR::TransformUtil::createConditionalAlternatePath(TR::Compilation* comp,
    cfg->addEdge(TR::CFGEdge::createEdge(ifBlock, thenBlock, comp->trMemory()));
    cfg->copyExceptionSuccessors(elseBlock, thenBlock);
    }
+
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
```

Comment:
Don't think `J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION` is visible in OMR, you'll need to use `OMR_GC_SPARSE_HEAP_ALLOCATION`. @LinHu2016 / @amicic, can you please confirm.